x-pack/plugins/maps/public/reducers/map/map.ts

```diff
@@ -131,7 +131,7 @@ export function map(state: MapState = DEFAULT_MAP_STATE, action: Record<string,
         ...state,
         mapState: {
           ...state.mapState,
-          mouseCoordinates: null,
```

Comment:
Change required by tslinter. mouseCoordinates is typed as 
```
mouseCoordinates?: {
    lat: number;
    lon: number;
  };
```
and tslinter was complaining about it getting set to null.